OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -750,6 +764,8 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
   bool is_amdgpu_sched_run_job_event = amdgpu_sched_run_job_ids_.contains(stream_id);
   bool is_dma_fence_signaled_event = dma_fence_signaled_ids_.contains(stream_id);
   bool is_callchain_sample = callchain_sampling_ids_.contains(stream_id);
+  bool is_tracepoint = stream_id_to_tracepoint_.contains(stream_id);
```

Comment:
Maybe `is_instrumented_tracepoint`? For example `task_rename`, `amdgpu_cs_ioctl_event`, are also tracepoints but don't fall into this category.

Reply:
Maybe make it even more clear and call it `is_user_instrumented_tracepoint`?

Reply:
Sure. Thank you a lot for the advice!